rollup.config.js

```diff
@@ -0,0 +1,56 @@
+import babel from "rollup-plugin-babel"
+import builtins from "rollup-plugin-node-builtins"
+import commonjs from "rollup-plugin-commonjs"
+import conditional from "rollup-plugin-conditional"
+import globals from "rollup-plugin-node-globals"
+import minify from "rollup-plugin-babel-minify"
+import pkg from "./package.json"
+import resolve from "rollup-plugin-node-resolve"
+import serve from "rollup-plugin-serve"
+
+let { buildTarget } = process.env
+
+if (typeof buildTarget === "undefined") {
+  buildTarget = "DEV"
+}
+
+const env = {
+  dev: buildTarget === "DEV",
+  prod: buildTarget === "PROD",
+}
+
+export default [
+  {
+    input: "browser.js",
+    output: [
+      {
+        file: pkg.main,
+        format: "iife",
+        name: "scratchblocks",
+        sourcemap: true,
+      },
+      {
+        file: pkg.module,
+        format: "esm",
+        sourcemap: true,
+      },
+    ],
+    plugins: [
+      resolve(),
+      commonjs(),
+      babel(),
+      globals(),
+      builtins(),
+      minify({
```

Comment:
Is it possible to make this plugin conditional on `env.prod`? This plugin is quite slow and it would be good to skip it for local testing -- rapid iteration is very important to me. :)

Reply:
Yeah that sounds like a great idea - I’ll make that change and push again tonight. I think it might be a good idea to merge this after that change and then tackle the translations in a separate issue and PR, so I’ll get on that on Monday :) Thanks!